governance/votes/Executive vote - August 10, 2022.md

```diff
@@ -0,0 +1,101 @@
+- The SocGen Vault will be activated.
+- A total of **121,808 DAI**, detailed below, will be transferred to multiple Recognized Delegates.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Huntingdon Valley (HVBank) Vault Drawdown
+
+Following the [addendum to the initial risk assessment](https://forum.makerdao.com/t/rwa009-hvbank-mip21-token-ces-domain-team-assessment/15861/8) after the [addition of RWA-009 (HVBank) as a new Vault Type](https://vote.makerdao.com/polling/QmQMDasC#vote-breakdown), **25,000,000 DAI** will be drawn down from the urn to the CryptoBroker by calling `drawFromRWA009Urn()` if this executive proposal passes.
+
+### MOMC Parameter Changes
+
+As per this [governance poll](https://vote.makerdao.com/polling/QmfMRfE4) on the latest MakerDAO Open Market Committee [proposal](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-28-july-2022/16895), the following parameter changes will be made if this executive proposal passes:
+
+**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Reductions**
+
+- Reduce **ETH-B** Stability Fee from 4% **to 3.75%**.
+- Reduce the **WSTETH-A** Stability Fee from 2.50% to **2.25%**.
+- Reduce the **WSTETH-B** Stability Fee from 0.75% to **0%**.
+- Reduce the **WBTC-B** Stability Fee from 4.00% to **3.75%**.
+
+**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Increases**
+
+- Increase the **GUNIV3DAIUSDC1-A** Stability Fee from 0.01% to **0.02%**.
+- Increase the **GUNIV3DAIUSDC2-A** Stability Fee from 0.05% to **0.06%**.
+- Increase the **UNIV2DAIUSDC-A** Stability Fee from 0.01% to **0.02%**.
+
+**[Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Increases**
+- Increase the **GUNIV3DAIUSDC2-A** Maximum Debt Ceiling from 1 billion DAI to **1.25 billion DAI**.
+- Increase the **GUNIV3DAIUSDC1-A** Maximum Debt Ceiling from 750 million DAI to **1 billion DAI**.
```

Comment:
I appreciate that this is my fault because I made the spreadsheet, but we've reversed the order of the GUNI vaults in the stab fee and line sections. Suggest making it consistent for readability reasons.

```suggestion
- Increase the **GUNIV3DAIUSDC1-A** Maximum Debt Ceiling from 750 million DAI to **1 billion DAI**.
- Increase the **GUNIV3DAIUSDC2-A** Maximum Debt Ceiling from 1 billion DAI to **1.25 billion DAI**.
```